# DP: gas: Recognize: AAarch64 ThunderX processor.

Index: b/gas/config/tc-aarch64.c
===================================================================
--- a/gas/config/tc-aarch64.c
+++ b/gas/config/tc-aarch64.c
@@ -7191,6 +7191,9 @@ static const struct aarch64_cpu_option_t
   {"exynos-m1", AARCH64_FEATURE (AARCH64_ARCH_V8,
 				 AARCH64_FEATURE_CRC | AARCH64_FEATURE_CRYPTO),
 				 "Samsung Exynos M1"},
+  {"thunderx", AARCH64_FEATURE (AARCH64_ARCH_V8,
+				AARCH64_FEATURE_CRC | AARCH64_FEATURE_CRYPTO),
+   "Cavium ThunderX"},
   /* The 'xgene-1' name is an older name for 'xgene1', which was used
      in earlier releases and is superseded by 'xgene1' in all
      tools.  */
Index: b/gas/doc/c-aarch64.texi
===================================================================
--- a/gas/doc/c-aarch64.texi
+++ b/gas/doc/c-aarch64.texi
@@ -59,6 +59,7 @@ on the target processor.  The following
 @code{cortex-a57},
 @code{cortex-a72},
 @code{exynos-m1},
+@code{thunderx},
 @code{xgene1},
 and
 @code{xgene2}.
